app/models/search_builder.rb

```diff
@@ -15,4 +15,30 @@ def only_lat_longs(solr_parameters)
     solr_parameters[:fq] ||= []
     solr_parameters[:fq] << "desc_metadata__latitude_teim:[* TO *]"
   end
+
+  def distance_query(solr_parameters)
+    solr_parameters[:fq] ||= []
+    if blacklight_params[:latitude] && blacklight_params[:longitude]
+      solr_parameters[:fq] << DistanceQueryGenerator.new(*blacklight_params.slice(:latitude, :longitude, :distance).values).to_s
```

Comment:
Splat operator. https://endofline.wordpress.com/2011/01/21/the-strange-ruby-splat/ Turns an array into a series of arguments.